quinn-proto/src/endpoint.rs

```diff
@@ -447,6 +463,35 @@ impl Endpoint {
         }
     }
 
+    /// Limiting the memory usage for items queued in the outgoing queue from endpoint
+    /// generated packets. Otherwise, we may see a build-up of the queue under test with
+    /// flood of initial packets against the endpoint. The sender with the sender-limiter
+    /// may not keep up the pace of these packets queued into the queue.
+    fn to_supresss_stateless_packets(&self) -> bool {
```

Comment:
nit: rename to `stateless_reset_packets_suppressed`?

Reply:
will do.